.github/workflows/ci_cron_daily.yml

```diff
@@ -38,6 +38,8 @@ jobs:
     with:
       submodules: false
       coverage: codecov
+      toxdeps: |
+        tox < 4
```

Comment:
Hopefully this is correct syntax. I am just guessing since OpenAstronomy did not provide example.